tests/test_client_methods.py

```
@@ -652,3 +660,75 @@ def test_list_tags_permission_denied(self, mock_zammad_api: Mock) -> None:
+        mock_instance.session.get.return_value = _mock_ticket_response(
+            {
+                "id": 1,
+                "group": "GSI Tech Team",
+                "state": "closed",
+                "priority": "2 normal",
+                "customer": "user@example.com",
+            }
+        )
+        mock_zammad_api.return_value = mock_instance
+
+        client = ZammadClient(url="https://test.zammad.com/api/v1", http_token="test-token")
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "http_token"

(S106)